general_rules/ContinueRules.tex

```diff
@@ -15,6 +15,7 @@ \subsection{Procedure}
 		\item The particular goal or desired result
 		\item How the action must be carried out (when necessary)
 		\item Details about how to interact with the robot (when necessary)
+		\item Detailed information to identify the objects (e.g. object name, color, location) for picking and placing objects.
```

Comment:
I think this is a very necessary change. But imo this needs to be a bit clearer. Does detailed information include one of the examples or all of them? 
I think the point of this change is so that teams don't use the Deus Ex Machina rules intended to skip manipulation to also skip perception. Now the requirements for this are different for different tests. In storing groceries the objects on the table are unknown so a robot showing me just a picture of an object it wants handed over is enough to show perception. In Serve Breakfast the robot know a bowl will be there so just showing a picture of the bowl does not show perception. Do you agree? Any idea how we can include this distinction. For example I think a robot trying to pick something up and failing and then asking for a handover always clearly shows perception.

Reply:
@johaq 
Thank you for your feedback, I just pushed a new commit.
I modified the example as well.
I have not mentioned particular task names or instructions because I think the rulebook should not force the team/robot to follow a particular procedure.